### PR TITLE
Re-create event PDFs

### LINKED ISSUE #1429: Update existing event PDFs

### PR DESCRIPTION
**Pull request by RickMohr**
* Data migration to clear all event PDF references
* When viewing event page, create PDF if it's missing

Fixes #1429

### CODE REVIEW

**Comment by RickMohr** [comment]
To test, apply migrations, go to an event page, wait for the "Download" link, and click it. You should see a large header containing the TreesCount logo.

File: src/nyc_trees/apps/event/migrations/0012_auto_20150424_1705.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def clear_event_pdfs(apps, schema_editor):
+    Event = apps.get_model("event", "Event")
+    for event in Event.objects.all():
+        event.map_pdf_filename = ''
+        event.save()
```
Comment (kdeloach) [inline src/nyc_trees/apps/event/migrations/0012_auto_20150424_1705.py]: Neat trick.

**Comment by steventlamb** [comment]
I added a commit in https://github.com/RickMohr/nyc-trees/pull/2 so that I could back out this migration before switching to another branch. I think it would be valuable to merge it.

**Comment by steventlamb** [comment]
I get this:

screenshot from 2015-04-27 14 11 41

any ideas what's causing this?

**Comment by steventlamb** [comment]
+1

I backed out the migration, ran static, chmodded my static/media vm dirs (probably just a vm quirk from running `debugcelery`), re-ran the migration, restarted celery and this works now.
screenshot from 2015-04-27 14 26 10

**Pull request merged**